authentication/account/worker/user_deactivation_worker_test.go

```diff
@@ -141,5 +203,36 @@ func mockRemoteCalls(userToDeactivate *account.User, identity account.Identity,
 		Post(fmt.Sprintf("/api/accounts/%s/deprovision_osio", identity.Username)).
 		MatchParam("authorization_username", config.GetOSORegistrationAppAdminUsername()).
 		// not checking token here. Refer to OSO Reg App Deactivation tests
-		Reply(200)
+		Reply(resStatus)
+}
+
+func mockCheCalls(config cheservice.Configuration) {
+	gock.New(config.GetCheServiceURL()).Reply(http.StatusNoContent)
+}
+
+func mockTenantCalls(config *configuration.ConfigurationData) {
+	gock.New(config.GetTenantServiceURL()).Reply(http.StatusNoContent)
+}
+
+func mockClusterCalls(config *configuration.ConfigurationData) {
+	gock.New(config.GetClusterServiceURL()).Reply(http.StatusOK).BodyString(`{
+		"data": [
+			{
+				"api-url": "starter-us-east-2a",
+				"app-dns": "b542.starter-us-east-2a.openshiftapps.com",
```

Comment:
I'd rather avoid using some real cluster URL in the tests. What was wrong with the `test-cluster`?

Reply:
Kept test data near to real one to get better idea for future reference.